### ISSUE TITLE
recursive submodules only work with new repositories

### ISSUE THREAD

**Issue by christian-rauch** [dirk-thomas/vcstool#86]
When switching the remote URL of an already existing git repo by forcing the import from a yaml file, e.g.
`vcs import --recursive --force --input repo.yaml`
git submodules are not checked out.

I manually need to delete the repo with submodules and repeat the command to also have submodules to be checked out.

**Comment by dirk-thomas** [comment]
The `--force` option is only relevant if the new repository URL is different than the already checked out one. In that case `--force` will first wipe the existing working copy and the following clone should work the same way as a fresh `import`. If the remote URL is different and you don't pass `--force` you should see the following error message for that repo:

> Path already exists and contains a different repository

In the case where the repository with the same URL is already cloned without the `--recursive` option and a second `import` with `--recursive` is invoked the recursive option is currently indeed not being taken into account. Please give #87 a try and comment if it works for you.

**Comment by christian-rauch** [comment]
Indeed, I got the `Path already exists ...` error and ran `vcs import < repo.yml --force` initially without the `--recursive` option. Using the `--recursive` option afterwards did not clone recursively.

The correct way would be to initially select both options: `vcs import < repo.yml --force --recursive`

I checked our your PR branch via:
`pip install git+https://github.com/dirk-thomas/vcstool.git@recursive_import_existing_wc`
and it solves the issue. I.e. I am able to use the `--recursive` option after the the repo has already been checked out.

Thanks.
